docs/kit/adoption-view/page_adoption-view.md

```diff
@@ -0,0 +1,48 @@
+---
+id: Adoption View
+title: Adoption View
+description: 'Connector Kit'
+sidebar_position: 1
+---
+
+The ConnectorKit provides a connector framework, based on the [Eclipse Dataspace Connector][edc-url] for sovereign, cross-enterprise data exchange.
+
+![EDC Overview](images/edc_overview.png)
```

Comment:
Your images should be editable somehow. Please be aware that there is currently an ongoing discusiion on aligning diagram drawing tool and were to put the original file.